### PR TITLE
add keyConcepts for Controlled Vocabulary

### PR DESCRIPTION
**Pull request by SrinivasanTarget**
@ioggstream @giorgialodi ping

### CODE REVIEW

File: VocabolariControllati/classifications-for-accommodation-facilities/accommodation-star-rating/accommodation-star-rating.ttl [hunk 1]
```diff
@@ -13,6 +13,7 @@
 @prefix foaf: <http://xmlns.com/foaf/0.1/> .
 @prefix vcard: <http://www.w3.org/2006/vcard/ns#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix ndc: <https://w3id.org/italia/onto/ndc-profile/> .
```
Comment (giorgialodi) [inline VocabolariControllati/classifications-for-accommodation-facilities/accommodation-star-rating/accommodation-star-rating.ttl:16]: @SrinivasanTarget the prefix of NDC is no longer this one but https://w3id.org/italia/onto/NDC/. It should be corrected in all CVs

**Review by giorgialodi — CHANGES_REQUESTED** [review]

**Comment by ioggstream** [comment]
@giorgialodi we need to regenerate all rdf+xml, jsonld files, right?

**Comment by giorgialodi** [comment]
> @giorgialodi we need to regenerate all rdf+xml, jsonld files, right?

hmmm, you are right because I see here the intervention only in the turtle files.

**Comment by ioggstream** [comment]
@giorgialodi I'm doing the fixes..

**Comment by ioggstream** [comment]
@SrinivasanTarget  I'll merge once you merge :P

**Comment by SrinivasanTarget** [comment]
@ioggstream your changes are merged. thanks.

**Review by giorgialodi — APPROVED** [review]
The prefix for NDC is not correct. I see a lot of other intervention not only for the ndc profile and keyConcepts here regarding spaces. @ioggstream what's that? Anyway, it seems to me ok anyway, so you can merge.

**Comment by ioggstream** [comment]
just trimming EOL/EOF spaces in edited files.

**Pull request merged**